apps/extension/src/state/password.ts

```diff
@@ -35,10 +40,38 @@ export const createPasswordSlice =
         void session.remove('hashedPassword');
         void local.remove('hashedPassword');
       },
+      clearSessionPassword: () => {
+        set(state => {
+          state.password.hashedPassword = undefined;
+        });
+        void session.remove('hashedPassword');
+      },
+      setCorrectPassword: () => {
+        set(state => {
+          state.password.isCorrectPassword = true;
+        });
+      },
       isPassword: async password => {
         const locallyStoredPassword = await local.get('hashedPassword');
         return locallyStoredPassword === repeatedHash(password);
       },
+      isUnlock: async password => {
```

Comment:
I would rename this to be `isPassword` and move that logic into here. I don't think that function is being used except here now.

Reply:
in the future, the ability to check the password when displaying the seed phrase will be required, as an example